### ISSUE TITLE
Capitalize default parameters "getMap", "getLegendGraphic" and "getFeatureInfo" in WMS configurator

### ISSUE THREAD

**Issue by antobenve** [unep-grid/mapx#928]
Some servers do not accept lowercase terms after "request=" in a WMS request. All servers seem to accept them in the capitalized form.
Therefore, capitalized terms should be favored and used when creating URLs in raster views:

getMap -> GetMap
getLegendGraphic ->  GetLegendGraphic 
getFeatureInfo -> GetFeatureInfo

Although the URLs for tiles and legend could be edited manually by the user in the raster configuration window - while the getFeatureInfo cannot - I think the user experience would be improved if the user could simply rely on the WMS configurator to produce functional URLs.

**Comment by fxi** [comment]
I found two cases in WMS tools where the parameter was not capitalized. There were no occurrences of `GetLegendGraphic` with missing capitalization. Did I missed something ?

```diff
diff --git a/app/src/js/wms/index.js b/app/src/js/wms/index.js
index 7d114c1a..4f25cf34 100644
--- a/app/src/js/wms/index.js
+++ b/app/src/js/wms/index.js
@@ -278,7 +278,7 @@ function urlTile(opt) {
     service: "WMS",
     version: "1.1.1",
     styles: "",
-    request: "getMap",
+    request: "GetMap",
     ZINDEX: 10,
     EXCEPTIONS: "application/vnd.ogc.se_blank",
     srs: "EPSG:3857",
@@ -300,7 +300,7 @@ function urlLegend(opt) {
     service: "WMS",
     version: "1.1.1",
     style: "",
-    request: "getLegendGraphic",
+    request: "GetLegendGraphic",
     layer: opt.layer,
     format: "image/png",
     tranparent: true,
```

**Comment by antobenve** [comment]
Maybe it was not clear, but I was especially referring to the WMS configuration tool. 

As you can see, clicking on the 
![image](https://github.com/user-attachments/assets/87bbe20c-5933-4506-8ae8-6ccd4de4d55c)
button produces URLs with no capitalization.
![image](https://github.com/user-attachments/assets/9dc14f1f-8be6-4e5f-8678-c4658136f636)

Of course, the idea is that attention should be paid to guarantee that the request is capitalized everytime the WMS request is effectively used in MapX, including GetFeatureInfo as well.

Am I missing something myself?

**Comment by antobenve** [comment]
I believe so too. I will test when possible.

**Comment by antobenve** [comment]
It is fixed in 1.13.12-alpha.4.
